code/modules/mob/inventory.dm

```diff
@@ -318,48 +318,32 @@
 		I.dropped(src, silent)
 	return TRUE
 
-//Outdated but still in use apparently. This should at least be a human proc.
-//Daily reminder to murder this - Remie.
-/mob/living/proc/get_equipped_items(include_pockets = FALSE)
-	return
+/**
+ * Used to return a list of equipped items on a mob; does not include held items (use get_all_gear)
+ *
+ * Argument(s):
+ * * Optional - include_pockets (TRUE/FALSE), whether or not to include the pockets and suit storage in the returned list
+ */
 
-/mob/living/carbon/get_equipped_items(include_pockets = FALSE)
+/mob/living/proc/get_equipped_items(include_pockets = FALSE)
 	var/list/items = list()
-	if(back)
-		items += back
-	if(head)
-		items += head
-	if(wear_mask)
-		items += wear_mask
-	if(wear_neck)
-		items += wear_neck
+	for(var/obj/item/I in contents)
+		if(I.item_flags & IN_INVENTORY)
+			items += I
```

Comment:
```suggestion
	for(var/obj/item/item_content in contents)
		if(item_content.item_flags & IN_INVENTORY)
			items += item_content
```
Use better var names than single upper-case letters.

Reply:
Thanks for letting me know I made the critical error of assuming this was correct due to the other procs in the file (big brained move assuming the other code in the file I was trying to fix bad code in was sound.)